File: Python For Everyone, Horstmann/Chapter2/P2.14.py
# Write a program that reads a number between 1,000 and 999,999 from the user and
# prints it with a comma separating the thousands.
# Here is a sample dialog; the user input is in color:
#   Please enter an integer between 1000 and 999999: 23456
#   23,456

userInput = str(int(input("Please enter an integer between 10000 and 99999: ")))

print(userInput[:2] + "," + userInput[-3:])

File: Big-Java-for-Everyone-Late-Objects-Horstmann/Chapter-2-Fundamental-Data-Types/P2_14.java
/*Write a program that reads a number between 1,000 and 999,999 from the user and
prints it with a comma separating the thousands.
Here is a sample dialog:
   Please enter an integer between 1000 and 999999: 23456
   23,456*/


import java.util.Scanner;


public class P2_14 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Integer between 1 000 and 999 999: ");
        String number = input.next();
        input.close();
        String postfix = number.substring(number.length() - 3);
        String prefix = number.substring(0, number.length() - 3);
        System.out.println(prefix + "," + postfix);
    }
}
